language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -1169,12 +1169,13 @@ public static String getBalaUriForPath(LanguageServerContext serverContext,
      * Returns the URI with bala scheme for the provided file path.
      *
      * @param filePath File path
+     * @param scheme URI Scheme
      * @return URI with bala scheme
```

Comment:
This is not necessarily bala scheme. Shall we change this?

Reply:
Done!